docs/support_playbook.md

```diff
@@ -128,7 +128,20 @@ If the current application status is `offer` use ChangeOffer service.
 
 This is possible via the support UI.
 
-### Revert an application choice to pending_conditions
+## Reverting an application choice to pending_conditions from recruited
+
+```ruby
+ac = ApplicationChoice.find(id)
+ac.update!(status: 'pending_conditions', recruited_at: nil, accepted_at: nil, audit_comment: 'Support request...')
+```
+Conditions can be added by creating a new OfferCondition object and then pushing it into the conditions collection, for example:
```

Comment:
Is it worth adding that the "pending" state is the DB default?

Reply:
Added, thanks